data/actions/actions.xml

```diff
@@ -42,6 +42,9 @@
 	<!-- Farmine -->
 	<action fromaid="9118" toaid="9121" script="farmine/elevatorLever.lua" />
 
+	<!-- Temple of Equilibrium -->
```

Comment:
I don't know how to use git shell I wasted some time to learn it but nothing happends >.<